### PR TITLE
Updating Information on Gas Refunds

### PR DESCRIPTION
**Pull request by ACStone-MTS**
This PR removes newly erroneous information about Casper not giving refunds for unspent CSPR.

As of the Peregrine update, this note is no longer relevant

### CODE REVIEW

**Comment by cspramit** [comment]
Note from 1.5.8 release  to extract information from - "An enhancement in Peregrine is where a refund system has been activated in the Casper protocol, meaning if a contract overpays, then 99% of the unspent fund will be refunded. This is achieved by changing the refund ratio in the configuration from 0% to 99%. This means that whatever was the payment amount made, minus whatever was the actual gas amount, which is the unspent amount, 99% of that will now be refunded and shown in the execution effects. "

**Review by ipopescu — APPROVED** [review]
LGTM. For CI/CD looks like some dependencies need to be updated. PR #121 from @deuszex takes care of the dependencies and needs to be merged.

```
error: package `bumpalo v3.15.4` cannot be built because it requires rustc 1.73.0 or newer, while the currently active rustc version is 1.68.0-nightly
Either upgrade to rustc 1.73.0 or newer, or use
cargo update -p bumpalo@3.15.4 --precise ver
```

File: docs/full-tutorial.md [hunk 1]
```diff
@@ -809,4 +807,4 @@ You can see that the name is `CasperTest` in this example.
  }
 }
 
-```
\ No newline at end of file
+```
```
Comment (cspramit) [inline docs/full-tutorial.md:812]: Are we just removing the previous text or adding new one too?

**Review by cspramit — APPROVED** [review]
lgtm

File: docs/full-tutorial.md (hunk 1 above)
Comment (ACStone-MTS) [reply docs/full-tutorial.md:812]: Just updated to include a note on the new system.

**Pull request merged**